ironic_client/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM docker.io/centos:centos7
+
+RUN yum install -y python-requests
+RUN curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - current-tripleo
+RUN yum install -y python-ironicclient python-openstackclient jq
+
+RUN echo "#!/usr/bin/bash" > /bin/runironic_client
```

Comment:
Why not just create the script here and `COPY` it during the build? This would create less layers

Reply:
Or you could end each line with a backslash to avoid the multiple RUN statements.

Reply:
Ack yeah good point, I just copied what we had already in ironic/Dockerfile, probably we can improve both